mine/macros/generic_tests/null_proportion.sql

```diff
@@ -0,0 +1,21 @@
+{% macro test_null_proportion(model) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
+{% set at_most = kwargs.get('at_most', kwargs.get('arg', 1)) %}
```

Comment:
Also, just in terms of name, what about max_threshold?

Reply:
max_threshold is more descriptive, but "null proportion is at most  x " sounds nicer?

Reply:
decided on "maximum"